### DISCUSSION TITLE
Having trouble ingesting a Google Doc

### DISCUSSION THREAD

**Discussion by bkiahstroud** [ManifoldScholar/manifold#3776] [Q&A] [answered]
Hello. I'm having issues ingesting a new Text from a Google Doc. I've ensured the doc is publicly accessible, however, I get the following error in the ingestion logs:

```
Connecting to Manifold websocket...
Successfully connected to websocket.
Downloading external resource: https://docs.google.com/document/d/1NYLzZW8EoKfaNJ_O-bV0BUSEIRBeJwS9W6PQcBTHPqs/edit?usp=sharing
Processing failed.

Unable to fetch ingestion source from https://docs.google.com/document/d/1NYLzZW8EoKfaNJ_O-bV0BUSEIRBeJwS9W6PQcBTHPqs/edit?usp=sharing
```

Does anyone have troubleshooting suggestions? Am I missing a step?

**Comment by tsmyre** [comment, marked as answer]
Is your instance integrated with Google Services? If not, then Manifold will not be able to secure Google Docs for ingestion (or use Google Sheets for the bulk upload of resources). Our documentation for that process lives here: https://manifoldscholar.github.io/manifold-docusaurus/docs/administering/configuring/integration_settings.

You will need admin credentials to access the pertinent views in Manifold, and you'll also need to create a project in the Google Developer Console.

**Reply by bkiahstroud** [reply]
Ah ha! I'm sure that's the step I'm missing, thank you for pointing me in the right direction.
